core/src/main/java/com/cognifide/aem/stubs/core/script/ConfigurableStubScriptManager.java

```diff
@@ -121,10 +131,20 @@ private void runAllEach(Resource resource, RunAllResult result, Stubs<?> runnabl
   @Override
   public Optional<Stubs<?>> findRunnable(String path) {
     return runnables.stream()
-      .filter(runnable -> wildcardMatch(path, format("%s/%s/**/*%s", getRootPath(), runnable.getId(), getExtension())))
+      .filter(runnable -> isStubScript(path, runnable) || isMappingFile(path, runnable))
       .findFirst();
   }
 
+  private boolean isStubScript(String path, Stubs<?> runnable) {
```

Comment:
DRY rule

Fully qualified : StubScriptManager.isStubScript.. You are repeating Stub... That's common java devs practice which I dislike very much. Context matters. Why not just isScript..